mod_app/templates/analysis_list.html

```diff
@@ -29,7 +31,8 @@ <h4 class="card__heading"><a href="{% url 'analysis_detail' pk=analysis.pk %}">{
                             <dd>{{film.title}}</dd>
                         {% endfor %}
                     </dl>
-                    <button href="{% url 'analysis_detail' pk=analysis.pk %}"> view full details </button>
+                    <button>
+                        <a href="{% url 'analysis_detail' pk=analysis.pk %}"> view full details </a></button>
```

Comment:
It's possible to click the button, but not the <a> element. Could you just style the link as a button, or do something with the button onclick event?